bcda/cclf/parser.go

```diff
@@ -55,25 +60,28 @@ func GetCSVMetadata(path string) (csvFileMetadata, error) {
 
 	acos, err := getACOConfigs()
 	if err != nil {
-		return csvFileMetadata{}, err
+		return csvFileMetadata{}, errors.New("Failed to load ACO configs")
+	}
+	if acos == nil {
+		return csvFileMetadata{}, errors.New("No ACO configs found.")
 	}
 
 	for _, v := range acos {
 		filenameRegexp := regexp.MustCompile(v.AttributionFile.NamePattern)
 		parts := filenameRegexp.FindStringSubmatch(path)
 		if len(parts) == v.AttributionFile.MetadataMatches {
 			metadata, err = validateCSVMetadata(parts)
+			log.API.Error("regex for aco (%s): %s", v.Model, v.AttributionFile.NamePattern)
```

Comment:
This is in development debugging (not expected to go to prod)?

Reply:
yup, same as above!